### ISSUE TITLE
Allow changing network setup during runtime

### ISSUE THREAD

**Issue by msmuenchen** [utmapp/UTM#6300]
Virtually all VM software allows changing the network backend on the fly during the machine runtime - both network mode (bridge, host-only, NAT) and connectivity itself.

At the moment, UTM only allows that when the machine is shut down, which is a bit annoying if one is installing Windows 11 as you need to disable networking to force it to create a local account.

**Comment by maeck** [comment]
Yes, please add this basic function! It drives me crazy, cause I often need to change the network interface.

**Comment by indiverbadal** [comment]
Yes, +1 for this feature request. This has been a thorny issue, otherwise UTM is perfect for me.
